articles/cross-origin-authentication/enabling-third-party-cookies.md

```diff
@@ -0,0 +1,31 @@
+---
+title: How to Enable Third-Party Cookies
+description: An explanation of how to enable third party cookies in certain web browsers
+---
+# How to Enable Third-Party Cookies
+
+In order to use a certain kind of authentication ([cross-origin authentication](/cross-origin-authentication)) to safely and securely sign in to various apps and services, a feature called "third-party cookies" is required. If you are using a particular web browser (such as Chrome and Opera on desktop or Android devices), signing in via cross-origin authentication can fail if third-party cookies are disabled in your browser.
```

Comment:
I would change the first sentence to `In order to use [cross-origin authentication](/cross-origin-authentication) for authentication you need to enable **third-party cookies**.`

Reply:
Maybe also add a short sentence on what third-party cookies are?

Reply:
Since this doc is for these two browsers only, I would change the last sentence to: `If you are using Chrome or Opera (on desktop or Android devices), signing in with cross-origin authentication can fail if third-party cookies are disabled in your browser.`

Also, is it `can fail` or `will fail`? Is it possible that it might work? If not change to `will`.